### PR TITLE
login page base

### PR DESCRIPTION
**Pull request by Earthii**
screen shot 2017-07-08 at 8 06 19 pm

### CODE REVIEW

**Comment by ghiobi** [comment]
Nice design

File: app/style.scss [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+}
+html, body, [ui-view]{
+  width:100%;
+  height:100%;
+}
+/*------------------+
+|     LOGIN PAGE    |
+ +------------------*/
+
+.loginPageContainr{
```
Comment (ghiobi) [inline app/style.scss]: I would put this in the separate file for the login style and then import it here. check other pr

File: app/style.scss [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+    display:flex;
+    flex-direction: column;
+    margin:auto;
+    background-color:#D3D6DB;
+    .logoContainer{
+      margin:auto;
+      background-color:yellow;
+    }
+    .loginButtonContainer{
+      margin:auto;
```
Comment (ghiobi) [inline app/style.scss]: generally, nesting CSS is bad due to specificity issues with it. but wtv

Comment (Earthii) [inline app/style.scss]: but the whole point of sass is for nesting lol

File: app/style.scss (hunk 1 above)
Comment (Earthii) [inline app/style.scss]: does webpack support different files now?

**Comment by Earthii** [comment]
made the changes

**Pull request merged**